### ISSUE TITLE
bloom_filter unit test fails

### ISSUE THREAD

**Issue by rsharris** [gmarcais/Jellyfish#91]
With the 2.2.6 release, I built this way:
   ./configure --prefix=somepath --without-int128
   touch aclocal.m4 configure Makefile.am Makefile.in
   make
   make install
   make check

The log output from make check says that tests/bloom_filter.sh failed. What can I do to gather more information about *how* it failed?

I've previously built jellyfish on three other machines, and didn't have this failure before. This is a new machine/server running ubuntu (Ubuntu 5.4.0). The other three machines were two Macs and some other variety of linux (that machine no longer exists, but I know it was not running ubuntu).  The mac builds were both 2.2.6, I'm not sure what the other linux build was (it was around July 2016).

I use jellyfish from the command line and also link the library with my own c++ program. Should I be worried about this failure?  If so, what can I do to figure out how to solve it?

**Comment by SethosII** [comment]
Same here, `make check` returns `FAIL: tests/bloom_filter.sh`. I also tried to circumvent the failing test with `GTEST_FILTER=-'*bloom*'` and `GTEST_FILTER=-'*Bloom*'` but they are still executed.

Built on a Ubuntu 16.04 with GCC 5.4.0.

See also https://github.com/hpcugent/easybuild-easyconfigs/issues/4141.

**Comment by SethosII** [comment]
The `test-suite.log` gave some insights:

```
=======================================
   jellyfish 2.2.6: ./test-suite.log
=======================================

# TOTAL: 13
# PASS:  8
# SKIP:  4
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: tests/bloom_filter
========================

./tests/bloom_filter.sh: line 11: bc: command not found
```

And indeed: `bc` wasn't installed. After the installation of `bc` it runs just fine, strange that `bc` isn't installed by default anymore in Ubuntu Server 16.04 (it was in 14.04).

**Comment by gmarcais** [comment]
Yes, the bloom filter unit test failing "randomly" is a misfeature (its a randomized data structure after all). I removed that part of the test. It will soon be part of the develop and main branches.

Regarding the dependency on bc in bloom_filter.sh. That's unfortunate, I did not think at the time that bc would ever be missing from a standard Linux install. I guess I was wrong. I now check in the script whether bc exists and only print a warning if it does not.
